Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift

```diff
@@ -110,6 +110,25 @@ private extension CardPresentPaymentStore {
             // https://stripe.dev/stripe-terminal-ios/docs/Enums/SCPPaymentIntentStatus.html#/c:@E@SCPPaymentIntentStatus@SCPPaymentIntentStatusRequiresCapture
             // TODO. Persist PaymentIntent, so that we can use it later to print a receipt
             // Deferred to https://github.com/woocommerce/woocommerce-ios/issues/3825
+
+            // Note: Here we transition from the Stripe Terminal Payment Intent
+            // to the WCPay backend managed Payment Intent, which we need
+            // to use to capture the payment.
+
+            self.remote.captureOrderPayment(for: siteID, orderID: orderID, paymentIntentID: intent.id) { result in
+                switch result {
+                case .success(let intent):
+                    if intent.status == .succeeded {
```

Comment:
This looks a bit more "swift" to me:

```
guard intent.status == .succeed else {
onFailure(...)
return
}

onCompletion(.sucess)
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/4004/commits/ad72e0190f0c72e064d51457bedb083f650d9496